web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -260,6 +262,110 @@ public String getFullRecord(String pidString, Model model, HttpServletRequest re
         return "fullRecord";
     }
 
+    /**
+     * JSON representation of full record
+     * Can remove the non-JSON version of the full record once all JSP files are ported over
+     * @param pidString
+     * @return
+     */
+    @GetMapping(path = "/{pid}/json", produces = APPLICATION_JSON_VALUE)
+    public @ResponseBody String handleRequest(@PathVariable("pid")String pidString) {
```

Comment:
I take it this is mostly a clone of the existing `getFullRecord` method except that it builds a map instead of a model? I guess we need to keep both until we have converted all full record pages

Reply:
Yeah, at the moment removing the original method breaks the work and file pages